### PR TITLE
Add documentation for PassTicket support

### PR DESCRIPTION
**Pull request by plavjanik**
This provides documentation for PassTicket support in APIML that is being developed.

#### Is there a related issue for this PR? 
Issue number: https://github.com/zowe/api-layer/pull/475

#### Developer's Certificate of Origin (DCO)

Signed-off-by: Petr Plavjanik <plavjanik@gmail.com>

### CODE REVIEW

**Comment by plavjanik** [comment]
Draft of release notes item:
- Zowe API Mediation Layer support single-sign-on for API services that accept PassTickets ([#456](https://github.com/zowe/api-layer/pull/465))

**Comment by Joe-Winchester** [comment]
@plavjanik - is this for 1.8 ?  Tks

**Comment by plavjanik** [comment]
@Joe-Winchester Hello Joe, we have just completed implementation and it is on the master branch of api-layer. We missed RC-1, should we try to include it to the next RC or wait for 1.9.0?

**Comment by Joe-Winchester** [comment]
@plavjanik - there are conflicts at the moment that need to be resolved before this PR can be merged.  Given it is doc I am happy for it to be merged into docs-staging.  There will be an rc2 cut later today so it'd be great if we can get the code and docs out

**Comment by plavjanik** [comment]
We have missed the RC2 because we had to fix a more critical problem first. We think it will be better to release PassTicket support with the next release 1.9.0 to get enough testing since it changes the way how authentication is handled in API Gateway even if PassTickets are not used.

File: docs/extend/extend-apiml/api-mediation-onboard-rest-service-direct-eureka-call.md [hunk 1]
```diff
@@ -129,21 +129,23 @@ where:
...
 
-```
+```xml
 <instance>
   <metadata>
       <apiml.catalog.tile.id>samples</apiml.catalog.tile.id>
       <apiml.catalog.tile.title>Sample API Mediation Layer Applications</apiml.catalog.tile.title>
       <apiml.catalog.tile.description>Applications which demonstrate how to make a service integrated to the API Mediation Layer ecosystem</apiml.catalog.tile.description>
       <apiml.catalog.tile.version>1.0.1</apiml.catalog.tile.version>
-      <apiml.service.title>Sample Service ©</apiml.service.title>
```
Comment (plavjanik) [inline docs/extend/extend-apiml/api-mediation-onboard-rest-service-direct-eureka-call.md]: I guess that has been used to test problems with such characters in the API Catalog UI but this just a copy of configuration in the documentation.

File: docs/extend/extend-apiml/api-mediation-onboard-rest-service-direct-eureka-call.md [hunk 2]
```diff
@@ -245,6 +248,46 @@ This prefix is used to differentiate the routes. This prefix must be provided ma
...
+        It is necessary to provide a service APPLID in the `apiml.authentication.applid` parameter.
+        
+        For more information, see [Enabling PassTicket creation for API Services that Accept PassTickets](api-mediation-passtickets.md)
+     
+     * **zosmf**
+     
+        This value specifies that a service accepts z/OSMF LTPA (Lightweight Third-Party Authentication).
+        This scheme should only be used for a z/OSMF service used by the API Gateway Authentication Service, and other z/OSMF services that are using the same LTPA key.
+        
+        For more information about z/OSMF Single Sign-on, see [Establishing a single sign-on environment](https://www.ibm.com/support/knowledgecenter/SSLTBW_2.4.0/com.ibm.zosmfcore.multisysplex.help.doc/izuG00hpManageSecurityCredentials.html)
```
Comment (plavjanik) [inline docs/extend/extend-apiml/api-mediation-onboard-rest-service-direct-eureka-call.md]: Thank you for adding this information!

**Comment by plavjanik** [comment]
Merged as part of #1059

**Pull request closed without merging**